qa/workunits/ceph-helpers-root.sh

```diff
@@ -28,6 +28,9 @@ function distro_version() {
 }
 
 function install() {
+    if [ $(distro_id) = "ubuntu" ]; then
+    sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
```

Comment:
Is the indentation right? Hard to know from github

Reply:
Same if indentation as in the rest of this file

Reply:
OK. For me it looks like lines 31 & 32 are identical.

Reply:
@ronen-fr it often looks off on Firefox, but Chrome shows the indentation correctly.